docs/policy.json.md

```diff
@@ -0,0 +1,248 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
+
+# Signature verification policy file format
+
+Signature verification policy files are used to specify policy, e.g. trusted keys,
+applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
```

Comment:
On Wed, Aug 10, 2016 at 04:53:27PM -0700, Miloslav Trmač wrote:
> > +The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
>
> It does seem odd to have a default file location in a library.  I’m
> thinking that having each user of the library decide on its own,
> very likely different, location, would be much worse.

Chipping in from the peanut gallery, I think the best solution to this
sort of thing is to use a set of standard rules for looking up config
files.  For example [1].

[1]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html